### PR TITLE
Fix build when there are spaces in the path

### PR DESCRIPTION
**Pull request by bingbing8**
Fix build when there are spaces in the path
https://github.com/PowerShell/Win32-OpenSSH/issues/810

### CODE REVIEW

File: contrib/win32/openssh/config.vcxproj [hunk 1]
```diff
@@ -1,243 +1,255 @@
...
+    </Link>
+    <PreBuildEvent>
+      <Command>powershell.exe -Executionpolicy Bypass -File "$(SolutionDir)config.ps1" -Config_h_vs "$(SolutionDir)config.h.vs" -Config_h "$(OpenSSH-Src-Path)config.h" -VCIncludePath "$(VC_IncludePath)" -OutCRTHeader "$(OpenSSH-Src-Path)contrib\win32\win32compat\inc\crtheaders.h"</Command>
+    </PreBuildEvent>
+    <PreBuildEvent>
+      <Message>Generate crtheaders.h and config.h</Message>
+    </PreBuildEvent>
+    <PostBuildEvent>
+      <Command>copy /Y "$(SolutionDir)install-ssh*ps1" "$(OutDir)
"
```
Comment (bagajjal) [inline contrib/win32/openssh/config.vcxproj]: ^M can be removed?

File: contrib/win32/openssh/config.vcxproj [hunk 2]
```diff
@@ -1,243 +1,255 @@
...
+copy  /Y "$(SolutionDir)uninstall-ssh*ps1" "$(OutDir)
```
Comment (bagajjal) [inline contrib/win32/openssh/config.vcxproj]: combine the below line.. same comment for multiple lines below..

**Review by bagajjal — APPROVED** [review]

File: contrib/win32/openssh/config.vcxproj (hunk 1 above)
Comment (bingbing8) [reply contrib/win32/openssh/config.vcxproj]: fixed

File: contrib/win32/openssh/config.vcxproj (hunk 2 above)
Comment (bingbing8) [reply contrib/win32/openssh/config.vcxproj]: fixed

**Pull request merged**